src/modules/hunting/cves.py

```diff
@@ -61,6 +62,16 @@ def __init__(self, binary_version):
         self.evidence = "kubectl version: {}".format(self.binary_version)
 
 
+""" kube-state-metrics CVE """
+
+class KubeStateMetricsVulnerability(Vulnerability, Event):
+    """"""
+    def __init__(self, version):
+        Vulnerability.__init__(self, KubectlClient, "kube-state-metrics Vulnerable To CVE-2019-17110", category=InformationDisclosure, vid="KVH051")
```

Comment:
shouldn't the vulnerable component be kubestatemetrics?

Reply:
Yes, correct. I missed this one.

Reply:
I am having a bit of difficulty understanding the component part, should I create it somewhere?

Reply:
Ideally you would want to declare the discovered kube-state-metrics as a `Service`, similar to `KubeDashboardEvent`, or `MetricsServer` as part of a `Discovery` class. Then you would set the component of the vulnerability as the kube-state-metrics service. @iYehuda am I correct?

Reply:
> Ideally you would want to declare the discovered kube-state-metrics as a `Service`, similar to `KubeDashboardEvent`, or `MetricsServer` as part of a `Discovery` class. Then you would set the component of the vulnerability as the kube-state-metrics service. @iYehuda am I correct?

Yes you are